keycloakClient/keycloakClient.go

```diff
@@ -535,96 +542,6 @@ func (c *Client) GetRealm(accessToken string, realmName string) (RealmRepresenta
 	return response, err
 }
 
-// GetClient get the representation of the client. idClient is the id of client (not client-id).
```

Comment:
Since i made a client applications file, I transferred over the existing methods.